3.11/appendix-deprecated.md

```diff
@@ -32,6 +32,9 @@ considered obsolete and may get removed in a future release. They are currently
 kept for backward compatibility. There are usually better alternatives to
 replace the old features with:
 
+- **Pregel Memory-Mapped Files**:
+  The option to use memory-mapped files as a backing storage for large datasets has been removed.
+
```

Comment:
Again, not sure if a removal notice goes into deprecation list. @Simran-B to clarify.